### PR TITLE
20 cards

### LINKED ISSUE #20: Improve Cards
- include several colum options

### PR DESCRIPTION
**Pull request by kesiah**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--edge-delivery-ubs-faweb-demo--netcentric.aem.live/cards
- After: https://20-cards--edge-delivery-ubs-faweb-demo--netcentric.aem.live/cards

Test URLs Text and image:
- Before: https://main--edge-delivery-ubs-faweb-demo--netcentric.aem.live/text-and-image
- After: https://20-cards--edge-delivery-ubs-faweb-demo--netcentric.aem.live/text-and-image

### CODE REVIEW

**Review by dragoss-nc — COMMENTED** [review]
Is this the only change or there are more to come?

File: blocks/cards/cards.css [hunk 1]
```diff
@@ -1,27 +1,52 @@
-.cards > ul {
+.cards__list {
+  display: grid;
+  grid-gap: 24px;
```
Comment (dragoss-nc) [inline blocks/cards/cards.css]: You can directly use `gap` here.

File: blocks/cards/cards.css [hunk 2]
```diff
@@ -1,27 +1,52 @@
...
+    overflow: hidden;
+  }
 }
 
-.cards .cards-card-body {
-  margin: 16px;
+.cards__card-body {
+  margin: 24px;
+
+  *:first-child {
```
Comment (dragoss-nc) [inline blocks/cards/cards.css]: I don't think you need `*` here. You could do directly `:first-child`.

**Review by dragoss-nc — APPROVED** [review]

File: blocks/cards/cards.css (hunk 1 above)
Comment (kesiah) [reply blocks/cards/cards.css]: true... I did not pay attention to it 😓

**Pull request merged**